doc/sphinxman/source/libint.rst

```diff
@@ -65,35 +65,38 @@ Installation
 
 **Binary**
 
-* .. image:: https://anaconda.org/psi4/libint/badges/version.svg
-     :target: https://anaconda.org/psi4/libint
+* .. image:: https://anaconda.org/psi4/libint2/badges/version.svg
```

Comment:
Are we the only ones who distinguish between "libint" and "libint2"? As far as I've seen on the [libint website](https://github.com/evaleev/libint), Ed just calls it "libint." Now that we don't have multiple versions of libint around, it might be cleaner and less confusing to drop the "2" from all references to the library.

Reply:
It's slightly complicated. To people, we can talk of the "libint" project, and I did keep the libint.rst file rather than making another. But CMake templates a lot on whatever is set as the project name and written as `<ProjectName>Config.cmake` (or `<projectname>-config.cmake`). Since that's Libint2, the cmake variables in this file had to change.

Reply:
You actually had that comment at anaconda. You're right that c-f uses `libint` for the packages. When psi can use a generic package, I can well see dropping "libint2" as the package label.

Reply:
Yeah, I mainly meant the conda package name, thanks!